door-open-service/node-backend/src/slackInterface.js

```diff
@@ -23,6 +23,19 @@ export default function (config, slackBot, sockServer) {
           "Consider dispatching a drone to pick up a human.");
 
     } else {
+
+      config.doorTimes = { "garage": 25 * 1000 };
```

Comment:
I don't think it's good to inject this into the config that this function receives. You should update the sample config files (see [this folder](https://github.com/FoundersFounders/door-services/tree/master/door-open-service/node-backend/config)) to include those `doorTimes` settings. I think it's reasonable to assume that if no time is set we should fallback to the previous behaviour (i.e. no lock).